html/Form.html

```diff
@@ -73,8 +73,8 @@
               <option value="Edit Suite 2 509">Edit Suite 2 509</option>
               <option value="Lab 502">Lab 502</option>
               <option value="Lobby/Hallway">Lobby/Hallway</option>
-              <option value="Other">Other</option>
-              <option value="Studio 21 North" class="hidden"></option>
+              <option value="OtherSelector">Other</option>
```

Comment:
Could you clarify this one?  It's not obvious why this change is needed to me, and the semantics are bothering me... the value of "OtherSelector" for an `<option>` within a `<select>` is not sitting right with me.

Reply:
Is the explaination in issue #13 not clear? I agree the language is not the best. Perhaps otherDisplay or temp could replace the blank string one and "other" could remain "other"

Reply:
is there no way to fix initializing the form in the javascript without changing the HTML?

Reply:
I did not see what the root issue was.  I see that there is a bug that causes the location drop down to not get set properly... can we just add something to the Form's show() function to initialize things?